src/Tools/dotnet-user-jwts/test/UserJwtsTests.cs

```diff
@@ -656,6 +656,21 @@ public void List_CanHandleProjectOptionAsPath()
         Assert.Contains(Path.Combine(projectPath, project), _console.GetOutput());
     }
 
+    [Fact]
+    public void List_CanHandleRelativePathAsOption()
+    {
+        var projectPath = _fixture.CreateProject();
+        var targetPath = Path.GetRelativePath(Path.GetTempPath(), projectPath);
+        var project = Path.Combine(projectPath, "TestProject.csproj");
+        Directory.SetCurrentDirectory(Path.GetTempPath());
```

Comment:
This confused me at first, it works because `GetTempPath` has the same relative path as another `GetTempPath` call. Could you instead call `GetTempPath` once, just to be a little cleaner?